eurotoken/README.MD

```diff
@@ -0,0 +1,42 @@
+# EuroToken - A Digital Euro
+_TODO: Write out in more detail about basics of EuroToken_
+
+## Web-of-Trust
+In standard operation, EuroToken communicates with a gateway controlled by authorities to verify the authenticity of transactions.
+However, EuroToken needs to be able to function in case of world-scale disasters and system outages, and thus in case of a gateway not being available for an extended period of time.
+Even more, EuroToken is also built such that it works in remote locations without a large peer-to-peer network.
+Not having a large network to confirm transactions with comes at the cost of induced risk, primarily due to the double spending problem.
+
+### Double spending
+Double spending is a problem that occurs when someone spends the same amount of money more than once.
+This problem is very prevalent with digital offline-capable currencies.
+
+The problem of double spending arises because digital currencies are not regulated by a central authority.
+This means that there is no one to verify that a transaction is valid.
+If no central authority is available to verify the validity of a transaction, then it is possible for someone to spend the same amount of money more than once.
+
+Double spending is a major problem for digital currencies, as it undermines their trustworthiness.
+If people cannot trust that a digital currency is not susceptible to double spending, then they are less likely to use it.
+
+Double spending in EuroToken occurs when a malicious user sends a transaction to a wallet, and then sends the same transaction to another wallet whilst the second receiver is not aware of the first transaction.
+
+#### Requirements for solution
+To adhere to the standard of EuroToken, the solution for double spending has to comply to the following requirements:
+* Fully offline capable
+* Completely distributed
+* Permissionless
+* Pseudo-anonymous
+* Independent of other authorities, such as banks
+
+### Double Spending mitigation
+To counter double spending, EuroToken attempts to create a [web-of-trust](https://en.wikipedia.org/wiki/Web-of-trust) to help verify the authenticity of transactions and users.
```

Comment:
This page does not exist, you probable mean: https://en.wikipedia.org/wiki/Web_of_trust